spaces/S000203/README.md

```diff
@@ -0,0 +1,5 @@
+---
+uid: S000203
+name: 3 points with a basis consisting of a singleton
```

Comment:
```suggestion
name: Three-point set with the basis $\{\{0\},X\}$
```

Follows the pattern of other spaces named as "three-point set"s. And avoids the subtlety of the definition of basis for when the whole space is the only neighborhood of a point.

Reply:
Makes sense. I will change the name of P204 as well.